chatui/libs/components/dropdown-menu/src/dropdown-menu.tsx

```diff
@@ -6,7 +6,21 @@ import { cn } from '@memgpt/utils';
 
 const DropdownMenu = DropdownMenuPrimitive.Root;
 
-const DropdownMenuTrigger = DropdownMenuPrimitive.Trigger;
```

Comment:
Let's not add styles here and use the asChild property instead. You can see how it's used to apply the drop down to a button in the RadixUI docs here: https://www.radix-ui.com/primitives/docs/components/dropdown-menu

Reply:
Saw you're already using asChild. Sorry about that! What was the reason for adding those styles?